service/models.py

```diff
@@ -73,10 +72,13 @@ def update(self, new_data):
         Updates a Inventory to the database
         """
         self.name = new_data.name or self.name
-        self.quantity = new_data.quantity or self.quantity
+        if hasattr(new_data, 'quantity'):
+            self.quantity = new_data.quantity
         self.reorder_quantity = new_data.reorder_quantity or self.reorder_quantity
         self.restock_level = new_data.restock_level or self.restock_level
         self.updated_at = datetime.utcnow()
+        if new_data.active is False:
+            self.active = new_data.active
```

Comment:
Why do we reassign only on cases of `False`?

Reply:
Because the OR statement written previously was unable to assign false if ```new_data.active``` is false